target/classes/public/index.html

```diff
@@ -359,6 +359,11 @@ <h1 class="title">Battleship</h1>
             description = "Click to toggle the difficulty of your opponent. Changing difficulty will reset the game!";
 
         break;
+        case "AircraftcarrierDiv":
+
+            description = "Select your desired ship and its orientation. Move your mouse over the tiles to get a preview, then click to place";
```

Comment:
should make the app much more user friendly especially for the people who maybe arent as computer fluent